app/views/providers/has_evidence_of_benefits/show.html.erb

```diff
@@ -8,14 +8,13 @@
       ) do |form| %>
 
     <%= form.govuk_collection_radio_buttons :has_evidence_of_benefit, # attribute name
-                                            yes_no_options(yes: Setting.enable_evidence_upload? ? t('.radio_hint_yes') : nil), # collection
+                                            yes_no_options(yes: t('.radio_hint_yes')), # collection
```

Comment:
do we need the collection comment?

Reply:
The comments at the end of each line here are the parameters as defined in the documentation of `govuk_collection_radio_buttons` - looking at the documentation [here](https://www.rubydoc.info/gems/govuk_design_system_formbuilder/GOVUKDesignSystemFormBuilder/Builder#govuk_collection_radio_buttons-instance_method) it's quite a complicated set of parameters to pass into this method, so I think it's useful to have the parameter names as a comment